static/src/js/types/sharedTypes.ts

```diff
@@ -216,6 +220,59 @@ export type MapGranule = {
   /** The spatial value for the granule */
   spatial: GeoJsonObject
 }
+export type ColormapScale = {
+  /** The scale object contains colors and labels */
+  scale: {
+    /** The colors in the colormap */
+    colors: string[]
+    /** The labels in the colormap */
+    labels: string[]
+  },
+  /** The classes object is not used */
+  classes?: undefined
+}
+
+export type ColormapClasses = {
+  /** The scale object is not used */
+  scale?: undefined
+  /** The classes object contains colors and labels */
+  classes: {
+    /** The colors in the colormap */
+    colors: string[]
+    /** The labels in the colormap */
+    labels: string[]
+  }
+}
+
+/** Colormap can have the scale or the classes object */
+export type Colormap = ColormapScale | ColormapClasses
+
+export type ImageryLayerItem = {
+  /** The product name */
+  product: string
+  /** The layer title */
+  title?: string
+  /** The colormap data for this layer */
+  colormap: Colormap
+  /** The opacity of the layer */
+  opacity: number
+  /** Whether the layer is visible */
+  isVisible: boolean
+  /** Layer properties */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any
```

Comment:
Do you need this?

Reply:
But we know what values might show up there, right? If we know what the data is that we are using then don't use `any`, it negates any value TypeScript is brining to the repo. That's why we don't ignore linting rules